pyproject.toml

```diff
@@ -1,7 +1,83 @@
 # Build-system section
 [build-system]
-requires = ["setuptools>=46.4.0", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling>=0.18.0"]
+build-backend = "hatchling.build"
+
+[project]
+name = "tuf"
+description = "A secure updater framework for Python"
+readme = "README.md"
+license = "MIT OR Apache-2.0"
+requires-python = "~=3.7"
+authors = [
+  { email = "theupdateframework@googlegroups.com" },
+]
+# Keep in sync with /docs/MAINTAINERS.txt
+maintainers = [
+  { name = "Justin Cappos", email = "jcappos@nyu.edu" },
+  { name = "Marina Moore", email = "mm9693@nyu.edu" },
+  { name = "Trishank Karthik Kuppusamy", email = "trishank@nyu.edu" },
+  { name = "Lukas Puehringer", email = "lukas.puehringer@nyu.edu" },
+  { name = "Joshua Lock", email = "jlock@vmware.com" },
+  { name = "Jussi Kukkonen", email = "jkukkonen@vmware.com" },
+]
```

Comment:
I think this is unneeded:
* it's more maintenance work
* I believe this will result in pypi showing all of our contact info in the project home page, I don't see that being useful: we don't really want anyone contacting _only_ maintainers (except for security issues but that process is documented in README).

We already have theupdateframework@googlegroups.com listed as "author", this sounds fine to me as it gives an email contact point.